rclcpp/include/rclcpp/strategies/static_memory_strategy.hpp

```diff
@@ -27,24 +27,46 @@ namespace memory_strategies
 
 namespace static_memory_strategy
 {
+  struct ObjectPoolBounds
+  {
+  public:
+    size_t max_subscriptions_;
+    size_t max_services_;
+    size_t max_clients_;
+    size_t max_executables_;
+    size_t pool_size_;
+
+    ObjectPoolBounds(size_t subs = 10, size_t services = 10, size_t clients = 10,
+      size_t executables = 1, size_t pool = 1024)
+    : max_subscriptions_(subs), max_services_(services), max_clients_(clients), max_executables_(
+        executables), pool_size_(pool)
+    {}
+  };
```

Comment:
Fixed in #82.

On Tue, Aug 11, 2015 at 9:23 AM, Dirk Thomas <notifications@github.com>
wrote:

> In rclcpp/include/rclcpp/strategies/static_memory_strategy.hpp
> <https://github.com/ros2/rclcpp/pull/81#discussion_r36765251>:
>
> > @@ -27,24 +27,46 @@ namespace memory_strategies
> >
> >  namespace static_memory_strategy
> >  {
> > +  struct ObjectPoolBounds
> > +  {
> > +  public:
> > +    size_t max_subscriptions_;
> > +    size_t max_services_;
> > +    size_t max_clients_;
> > +    size_t max_executables_;
> > +    size_t pool_size_;
> > +
> > +    ObjectPoolBounds(size_t subs = 10, size_t services = 10, size_t clients = 10,
> > +      size_t executables = 1, size_t pool = 1024)
> > +    : max_subscriptions_(subs), max_services_(services), max_clients_(clients), max_executables_(
> > +        executables), pool_size_(pool)
> > +    {}
> > +  };
>
> Style warning:
> http://ci.ros2.org/job/ros2_batch_ci_osx/100/testReport/junit/rclcpp/uncrustify/include_rclcpp_strategies_static_memory_strategy_hpp/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ros2/rclcpp/pull/81/files#r36765251>.
>